src/index.js

```diff
@@ -66,26 +66,32 @@ const App = () => {
                     {
                       text: <Text message='navigation.dashboard' />,
                       action: '/',
+                      iconType: 'home',
                     },
                     {
                       text: <Text message='navigation.lessons' />,
                       action: '/lessons',
+                      iconType: 'list-ul',
                     },
                     {
                       text: <Text message='navigation.instructors' />,
                       action: '/instructors',
+                      iconType: 'user',
                     },
                     {
                       text: <Text message='navigation.guide' />,
                       action: guideUrl,
+                      iconType: 'question',
                     },
                     {
                       text: <Text message='navigation.chat' />,
                       action: chatUrl,
+                      iconType: 'slack',
                     },
                     {
                       text: <Text message='navigation.logout' />,
                       action: logout,
+                      iconType: 'sign-out',
                     },
```

Comment:
Adds icons to navigation items. Vojta wants to customize these in the future.

![image](https://cloud.githubusercontent.com/assets/5497885/24430153/7d7ac7d0-13d2-11e7-83c1-b8cbe350c396.png)